docs/docs/tutorial/jsonnet.mdx

```diff
@@ -286,12 +286,15 @@ clusters just work.
 This allows us to make sure that you will never accidentally apply to the wrong
 cluster.
 
-> **Note**: Tanka won't create the namespace for you. Either include it in
-> Jsonnet, or with the following command:
+> **Note**: Tanka won't create the namespace for you -- you need to include it in
+> Jsonnet. Alternatively, you can create the namespace manually:
```

Comment:
would be great to show how you *do* create a namespace via Jsonnet. It isn't particularly hard. I think this will do it:

```
{
  my_namespace: $.core.v1.namespace.new('my-namespace'),
}

Reply:
I added the snippet to the docs.